docs/index.md

```diff
@@ -1,36 +1,62 @@
 ---
 layout: ""
-page_title: "Provider: kentik-cloudexport"
+page_title: "Provider: Kentik CloudExport"
```

Comment:
Changing the name to CamelCase is undesirable, because the name `terraform` registry refers to the provider as `kentik-cloudexport`. It should be exactly this string regardless of where it is included (if some doc linters disagree, just silence them).